test/gpu/native/studies/sort/radixSortGpu.chpl

```diff
@@ -125,18 +223,156 @@ module RadixSort {
     if scanTest then
       writeln(" Count: ", count);
 
-
-    if scanType == "hillisSteele" then
-      hillisSteeleScan(count, buckets);
-    else if scanType == "blelloch" then
-      blellochScan(count, buckets);
-    else if scanType == "serial" then
-      serialScan(count, buckets);
+    if parallelScan then parallelArrScan(count); else arrScan(count);
 
     if scanTest then
       writeln("Pf sum: ",count);
   }
 
+
+  proc parallelCount2D(ref counts: [?D], ref inputArr: [] uint, const exp: int, const bitMask: int, const buckets: int){
```

Comment:
counts is an array of arrays, correct?

Reply:
Yes!

Reply:
Offhand: first time I am seeing array-of-arrays on GPU. Glad to see they work.